render.yaml

```
@@ -0,0 +1,22 @@
+# Render blueprint — deploys the Kryptos FastAPI backend as a Docker web service.
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)